Makefile

```diff
@@ -62,8 +62,8 @@ win_xcalib: xcalib.c
 	$(CC) $(CFLAGS) -mwindows -lm resource.o -o xcalib xcalib.o
 
 install:
-	cp ./xcalib $(DESTDIR)/usr/local/bin/
-	chmod 0644 $(DESTDIR)/usr/local/bin/xcalib
+	cp -pv ./xcalib $(DESTDIR)/usr/local/bin/
+#	chmod 0755 $(DESTDIR)/usr/local/bin/xcalib
```

Comment:
Can be removed entirely IMO.

Reply:
it's not clear why someone would need to fiddle with those bits after building, testing, and installing.

Reply:
Then I would prefer a remove of this line before merge.